### DISCUSSION TITLE
Intercompartment Concentrations

### DISCUSSION THREAD

**Discussion by pburkat** [Open-Systems-Pharmacology/Forum#995] [Questions & Problems]
Hello, is it possible to observe the concentration change of a compound in one compartment with respect to another? I am interested in knowing what the concentration of a compound would be in the unbound interstitial compartment of the brain as it diffuses from the brain tissue compartment only.

**Comment by StephanSchaller** [comment]
Hi @pburkat,

I am not sure if I got you right, but you can directly output the brain intracellular unbound concentrations:
![image](https://user-images.githubusercontent.com/25477643/138042221-4b65298c-75d6-4fb3-a6aa-592f7f11cb2a.png)

**Reply by pburkat** [reply]
Actually it's not the intracellular compartment. It would be the unbound interstitial compartment still, but what fraction of it is coming from the tissue compartment?

**Reply by pburkat** [reply]
For example, a compound that is located in the tissue lipid membrane, but has to access it's receptor from the extracellular side of a neuron, so it will leave the lipid compartment slowly and relocate to the interstitial compartment with a certain rate and magnitude.
